### PR TITLE
Fix bugs with abandoned promises.

### PR DESCRIPTION
**Pull request by beefo**

### CODE REVIEW

File: test/node/nocks/nock.js [hunk 1]
```diff
@@ -74,8 +80,14 @@ get
     })
     .reply(200, {
         error: {
-            name: 'InvalidRequest',
-            message: 'Invalid request information entered'
+            code: -32000,
+            name: "JSONRPCError",
+            message: "The method 'Geet' could not be found. Verify the method name and ensure all method parameters are included.",
```
Comment (ravneet-s) [inline test/node/nocks/nock.js:85]: The method "Get"

File: test/node/node-Authenticate-Invalid-Host.spec.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+require('./nocks/nock');
+require('source-map-support').install();
+
+/**
+ *  Tests the core functionality of failing cases
+ *  Tests failures against call -> Call will be the failing point of most requests
+ *  via bad args or credentials
+ */
+describe('User loads GeotabApi node module and triggers an AxiosError', async () => {
+    it('Propigates exception from Axios that URL not found', async () => {
```
Comment (ravneet-s) [inline test/node/node-Authenticate-Invalid-Host.spec.js]: "Propagates"

File: test/web/serverSetup.js [hunk 3]
```diff
@@ -34,65 +34,79 @@ module.exports = async function(){
...
                         // Poorly formed request tests
-                        payload = { result: {
-                            name: "InvalidCall",
-                            message: "Bad info entered"
-                        }}
+                        payload = {
+                            error: {
+                                code: -32000,
+                                name: "JSONRPCError",
+                                message: "The method 'Geet' could not be found. Verify the method name and ensure all method parameters are included.",
```
Comment (ravneet-s) [inline test/web/serverSetup.js:66]: "Get"

File: test/node/nocks/nock.js (hunk 1 above)
Comment (beefo) [reply test/node/nocks/nock.js:85]: Hard to tell from the diff (and maybe by the method name choosen) but this is intentional. The method called is 'Geet'.

File: test/web/serverSetup.js (hunk 3 above)
Comment (beefo) [reply test/web/serverSetup.js:66]: Hard to tell from the diff (and maybe by the method name choosen) but this is intentional. The method called is 'Geet'.

**Pull request merged**